Source/ALSV4_CPP/Private/Character/AI/ALS_BTTask_GetRandomLocation.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2020, Doga Can Yanikoglu
+// @author: Jens Bjarne Myhre
+
+#include "Character/AI/ALS_BTTask_GetRandomLocation.h"
+#include "AIController.h"
+#include "BehaviorTree/BlackboardComponent.h"
+#include "BehaviorTree/Blackboard/BlackboardKeyType_Vector.h"
+
+UALS_BTTask_GetRandomLocation::UALS_BTTask_GetRandomLocation()
+{
+	NodeName = "Get Random Location";
+
+	BlackboardKey.AddVectorFilter(this, GET_MEMBER_NAME_CHECKED(UALS_BTTask_GetRandomLocation, BlackboardKey));
+}
+
+EBTNodeResult::Type UALS_BTTask_GetRandomLocation::ExecuteTask(UBehaviorTreeComponent& OwnerComp, uint8* NodeMemory)
+{
+	if(UBlackboardComponent* Blackboard = OwnerComp.GetBlackboardComponent())
+	{
+		if (AAIController* Controller = OwnerComp.GetAIOwner())
+		{
+			if(APawn* Pawn = Controller->GetPawn())
```

Comment:
To keep our coding style consistent with rest of the source code, can you define variables out of the if condition? That's applicable for other similar cases in the PR.